lib/kitchen/provisioner/chef_base.rb

```diff
@@ -47,18 +48,6 @@ class ChefBase < Base
         providers/**/* recipes/**/* resources/**/* templates/**/*
       ].join(",")
 
-      default_config :chef_metadata_url do |provisioner|
-        provisioner.default_windows_chef_metadata_url if provisioner.windows_os?
-      end
-
-      default_config :chef_omnibus_root do |provisioner|
```

Comment:
It's supported in mixlib-install but not implemented in test-kitchen right now; I'll make it so.